cpp/tests/strings/pad_tests.cu

```diff
@@ -120,3 +121,38 @@ TEST_F(StringsPadTest, ZFill)
         thrust::make_transform_iterator( h_expected.begin(), [] (auto str) { return str!=nullptr; }));
     cudf::test::expect_columns_equal(*results,expected);
 }
+
```

Comment:
Why is this file called `pad_tests` if it's testing wrap?

Reply:
Because more than just `wrap` is being tested in that file. The file existed already, I didn't create it.

It follows the legacy structure of collecting all tests pertaining to padding. Including `wrap`.